.drone.yml

```
@@ -0,0 +1,83 @@
+clone:
+  git:
+    image: registry.fpfis.eu/drone-plugins/git:next
+    when:
+      event: [ push, pull_request, tag ]
+  git:
+    image: registry.fpfis.eu/drone-plugins/git
+    when:
+      event: [ deployment ]
+
+workspace:
+  base: /test
+  path: oe_starter_content
+
+services:
+  web:
+    image: fpfis/httpd-php-ci:${PHP_VERSION}
+    environment:
+      - DOCUMENT_ROOT=/test/oe_starter_content
+      - COMPOSERVER=--2
+  mysql:
+    image: percona/percona-server:5.6
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+
+pipeline:
+  composer-install-highest:
+    group: prepare
+    image: fpfis/httpd-php-ci:${PHP_VERSION}
+    volumes:
+      - /cache:/cache
+    commands:
+      - composer self-update --2
+      - composer install --ansi --no-progress
```

lets leave the suggests out 
```suggestion
      - composer install --ansi --no-suggest --no-progress
```

---

Suggest is deprecated in composer 2 and will be removed from 3, so we don't use it anymore